usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -132,6 +157,7 @@ public boolean isDeprecated() {
     @Override
     public String toString() {
         return Objects.toStringHelper(this)
+                .add("tags", tags)
```

Comment:
I thought it would be useful to have it in the `toString()` method but I don't have a strong feeling about it. I can remove it if you want.